dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/preheat/supplier/FileResourceSupplierTest.java

```diff
@@ -53,148 +59,223 @@
     {
-        DataValue fileResourceDataValue = new DataValue();
-        fileResourceDataValue.setDataElement( MetadataIdentifier.ofUid( FILE_RESOURCE_DATA_ELEMENT_UID ) );
-        fileResourceDataValue.setValue( FILE_RESOURCE_UID );
 
-        DataValue emptyFileResourceDataValue = new DataValue();
-        emptyFileResourceDataValue.setDataElement( MetadataIdentifier.ofUid( EMPTY_FILE_RESOURCE_DATA_ELEMENT_UID ) );
-        emptyFileResourceDataValue.setValue( "" );
+        return Attribute.builder()
+            .attribute( MetadataIdentifier.ofUid( uid ) )
+            .valueType( ValueType.FILE_RESOURCE )
+            .value( value )
+            .build();
+    }
 
-        DataValue nullFileResourceDataValue = new DataValue();
-        nullFileResourceDataValue.setDataElement( MetadataIdentifier.ofUid( NULL_FILE_RESOURCE_DATA_ELEMENT_UID ) );
-        nullFileResourceDataValue.setValue( null );
+    private TrackedEntity trackedEntity( Attribute... attributes )
+    {
+        return TrackedEntity.builder()
+            .attributes( attributes( attributes ) )
+            .build();
+    }
 
-        DataValue numericDataValue = new DataValue();
-        numericDataValue.setDataElement( MetadataIdentifier.ofUid( NUMERIC_DATA_ELEMENT_UID ) );
-        numericDataValue.setValue( "3" );
+    private Enrollment enrollment( Attribute... attributes )
+    {
+        return Enrollment.builder()
+            .attributes( attributes( attributes ) )
+            .build();
+    }
 
-        Event event = new Event();
-        event.setDataValues(
-            Sets.newHashSet( fileResourceDataValue, emptyFileResourceDataValue, nullFileResourceDataValue,
-                numericDataValue ) );
+    private List<Attribute> attributes( Attribute[] attributes )
+    {
+        List<Attribute> attrs = new ArrayList<>();
+        for ( Attribute at : attributes )
+        {
+            attrs.add( at );
+        }
+        return attrs;
+    }
 
-        return event;
+    private DataElement numericDataElement( String uid )
+    {
+        DataElement element = createDataElement( 'A' );
+        element.setUid( uid );
+        element.setValueType( ValueType.NUMBER );
+        return element;
+    }
+
+    private DataElement fileDataElement( String uid )
+    {
+        DataElement element = createDataElement( 'A' );
+        element.setUid( uid );
+        element.setValueType( ValueType.FILE_RESOURCE );
+        return element;
+    }
+
+    private Event event( DataValue... dataValues )
+    {
+        return Event.builder()
+            .dataValues( dataValues( dataValues ) )
+            .build();
+    }
+
+    private Set<DataValue> dataValues( DataValue[] dataValues )
+    {
+        Set<DataValue> dvs = new HashSet<>();
+        for ( DataValue dv : dataValues )
+        {
+            dvs.add( dv );
+        }
+        return dvs;
+    }
+
+    private DataValue dataValue( String uid, String value )
+    {
+
+        return DataValue.builder()
```

Comment:
also here remove empty lines

Reply:
either we are using a poor formatter or its not well configured if we still have to talk about formatting even when its used 😢 
removed them 👍🏻